robottelo/virtwho_utils.py

```diff
@@ -208,7 +218,7 @@ def _get_hypervisor_mapping(logs, hypervisor_type):
     """Analysing rhsm.log and get to know: what is the hypervisor_name
     for the specific guest.
     :param str logs: the output of rhsm.log.
-    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt
+    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt. ahv
```

Comment:
```suggestion
    :param str hypervisor_type: esx, libvirt, rhevm, xen, libvirt, kubevirt, ahv
```

Reply:
> 
Have already submit the update. Please confirm it thanks.